cachito/workers/scm.py

```diff
@@ -62,15 +63,50 @@ def _reset_git_head(self, repo):
                 f'of "{self.ref}" is valid.'
             )
 
+    def _verify_archive(self):
+        """
+        Verify the archive containing the git repository.
+
+        :raises CachitoError: if 'git fsck' fails for the extracted sources
+        """
+        log.debug("Verifying the archive at %s", self.sources_dir.archive_path)
+        if not os.path.exists(self.sources_dir.archive_path) or not tarfile.is_tarfile(
+            self.sources_dir.archive_path
+        ):
+            raise CachitoError(f"No valid tar archive at {self.sources_dir.archive_path}")
```

Comment:
```suggestion
            log.error(f"No valid archive found at %s, {self.sources_dir.archive_path}")
            raise CachitoError(f"No valid archive found at {self.sources_dir.archive_path}")
```
I think we have only been using the word just `archive` as it is assumed to be a tar only, and the word 'tar' would be more for developers than the users?

Also, should we log the error as well?

Reply:
Changed the exception message.

Do we need to repeat ourselves in the logs here? There is no additional information being added in the log message and I do not see how this exception message would be missing from the logs.

Reply:
It wouldn't be missed from the visibility in `/<request-id>/logs` url content as that's how the request would fail and it'd point out to this line at the end of the logs but my suggestion was more to be consistent with logging throughout the module. One benefit I could see is, if in the future we are querying from kibana or someplace else through logs, this could be helpful. One such example can be found at https://github.com/release-engineering/cachito/blob/master/cachito/workers/pkg_managers/general.py#L58